src/components/App/index.js

```diff
@@ -48,6 +49,7 @@ const App = ({
       <hr />
       <section id="rules" className="Rules">
```

Comment:
U have className Rules used 2 time. Here and in a component. Is that intentionally?